### PR TITLE
Apply code style fixes

### PR DESCRIPTION
**Pull request by glensc**
Minor code style fixes done with help of [php-cs-fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer).

### CODE REVIEW

**Comment by glensc** [comment]
1. not directly. i was about to make changes, but noticed `is_null` in place
2. i don't use the tool. shouldn't travis complain on this?

**Comment by aik099** [comment]
> 1. not directly. i was about to make changes, but noticed `is_null` in place

I see.

> 2. i don't use the tool. shouldn't travis complain on this?

It's not doing that currently. It makes sense to do only, when we fix all existing issues to avoid all new build be broken by default.

**Comment by glensc** [comment]
checked `vendor/bin/phpcs`, the changes here are not related.

**Comment by aik099** [comment]
Maybe can combine all CS fixes into single PR? I've just updated `composer.json` and `composer.lock` to ensure that used CS is compatible with latest PHP_CodeSniffer version.

PHP_CodeSniffer also does support auto-fixing of most, but not all issues found.

If you're using PhpStorm, then you configure it to automatically highlight problematic places.

We're not using Scrutinizer CI or otherwise it will be showing CS issues online as well.

**Comment by glensc** [comment]
You can squash merge from GitHub web UI if you have enabled that in project. If that's what you mean by single commit.

**Comment by glensc** [comment]
At this moment, I do not plan to make more CS fixes.

File: src/Jira/Api.php [hunk 1]
```diff
@@ -741,16 +741,15 @@ public function api(
 				}
 			}
 
-			if ( $return_as_array ) {
+			if ($return_as_array) {
```
Comment (aik099) [inline src/Jira/Api.php]: Please restore spaces, that were removed.

---

As you can see from majority of above changed IFs the condition is wrapped in spaces.

File: src/Jira/Api.php [hunk 2]
```diff
@@ -741,16 +741,15 @@ public function api(
...
 			}
-			else {
-				return new Result($json);
-			}
-		}
-		else {
-			return false;
+
+			return new Result($json);
+
```
Comment (aik099) [inline src/Jira/Api.php]: Please remove an empty line.

---

No need to add an empty line before closing `}`.

**Comment by aik099** [comment]
Merging, thanks @glensc .

**Pull request merged**